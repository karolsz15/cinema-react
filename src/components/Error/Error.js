import React from 'react';

const Error = () => (
    <div class="main">
        <div class="contact-content">
			<div class="logo">
				<a href="/"><img src="images/logo4.png" alt="" /></a>
				<p>Ticket booking app</p>
			</div>
			<div class="clearfix"></div>
		</div>

        <div class="contact-content">
            <div class="main-contact">
                <img class="errorImg" alt="Error message image" src="https://firebasestorage.googleapis.com/v0/b/karol-cinema.appspot.com/o/error404.jpg?alt=media&token=cbb825e6-bc0c-408f-a317-1552486bd90e"></img>
            </div>
        </div>
    </div>
);

export default Error;