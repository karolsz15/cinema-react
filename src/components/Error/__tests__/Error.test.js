import React from 'react';
import { shallow } from 'enzyme';
import Error from '../Error';

let wrapper = shallow(<Error />);

describe('Error component', () => {
  it('should render the Error correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});