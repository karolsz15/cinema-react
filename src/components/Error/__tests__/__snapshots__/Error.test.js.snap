// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render the Error correctly 1`] = `
<div
  className="main"
>
  <div
    className="contact-content"
  >
    <div
      className="logo"
    >
      <a
        href="/"
      >
        <img
          alt=""
          src="images/logo4.png"
        />
      </a>
      <p>
        Ticket booking app
      </p>
    </div>
    <div
      className="clearfix"
    />
  </div>
  <div
    className="contact-content"
  >
    <div
      className="main-contact"
    >
      <img
        alt="Error message page not found"
        className="errorImg"
        src="https://firebasestorage.googleapis.com/v0/b/karol-cinema.appspot.com/o/error404.jpg?alt=media&token=cbb825e6-bc0c-408f-a317-1552486bd90e"
      />
    </div>
  </div>
</div>
`;
